docs/translations/api-docs/data-grid/grid-list-col-def.json

```
@@ -0,0 +1,16 @@
+{
+  "interfaceDescription": "Column Definition interface used for the list view column.",
+  "propertiesDescriptions": {
+    "field": {
+      "description": "The column identifier. It&#39;s used to map with GridRowModel values."
+    },
+    "align": { "description": "Allows to align the column values in cells." },
```

"allows to" is a Frenchism that's grammatically incorrect in English ("allows" requires a subject, e.g. "allows _you_ to"), and it pops up often in the MUI docs—it's usually best replaced with "makes it possible to."

```suggestion
    "align": { "description": "Makes it possible to align column values in cells." },
```

---

Alternatively you could choose a more concise structure:

```suggestion
    "align": { "description": "Align column values in cells." },
```